scrapy_zyte_api/handler.py

```diff
@@ -23,6 +23,27 @@
 logger = logging.getLogger(__name__)
 
 
+def _body_max_size_exceeded(
+    body_size: int,
+    warnsize: Optional[int],
+    maxsize: Optional[int],
+    request_url: str,
+) -> bool:
+    if warnsize and body_size > warnsize:
+        logger.warning(
+            f"Actual response size {body_size} larger than "
+            f"download warn size {warnsize} in request {request_url}."
+        )
+
+    if maxsize and body_size > maxsize:
+        logger.warning(
+            f"Cancelling download of {request_url}: actual response size "
```

Comment:
This message seems to be incorrect: we're not cancelling download anymore (as download already happened - we can't cancel it, unlike Scrapy's default download handler), we're dropping the response.

Reply:
Ah, of course, that's more correct - fixed!